src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -2473,11 +2474,12 @@ Real64 CalcASHRAESimpExtConvectCoeff(int const Roughness,       // Integer index
     Real64 CalcASHRAESimpExtConvectCoeff;
 
     // FUNCTION PARAMETER DEFINITIONS:
-    static Array1D<Real64> const D(6, {11.58, 12.49, 10.79, 8.23, 10.22, 8.23});
-    static Array1D<Real64> const E(6, {5.894, 4.065, 4.192, 4.00, 3.100, 3.33});
-    static Array1D<Real64> const F(6, {0.0, 0.028, 0.0, -0.057, 0.0, -0.036});
+    static Array1D<Real64> const D({0, 5}, {11.58, 12.49, 10.79, 8.23, 10.22, 8.23});
+    static Array1D<Real64> const E({0, 5}, {5.894, 4.065, 4.192, 4.00, 3.100, 3.33});
+    static Array1D<Real64> const F({0, 5}, {0.0, 0.028, 0.0, -0.057, 0.0, -0.036});
```

Comment:
I guess maybe there was a reason these couldn't just be `std::array`?  I'll probably find the reason as I read along.

Reply:
I'll change these to std::arrays in the next PR. Thanks for spotting these!